src/com/koxudaxi/ruff/RuffConfigService.kt

```diff
@@ -1,13 +1,11 @@
 package com.koxudaxi.ruff
 
-import com.intellij.openapi.components.PersistentStateComponent
-import com.intellij.openapi.components.Service
-import com.intellij.openapi.components.State
-import com.intellij.openapi.components.Storage
+import com.intellij.openapi.components.*
 import com.intellij.openapi.project.Project
 import com.intellij.util.xmlb.XmlSerializerUtil
 import org.jetbrains.annotations.SystemDependent
 
+
 @State(name = "RuffConfigService", storages = [Storage("ruff.xml")])
 @Service(Service.Level.PROJECT)
 class RuffConfigService : PersistentStateComponent<RuffConfigService> {
```

Comment:
If I add the `project: Project` in the argument, the service raises a null point exception.
I had no choice but to create RuffMacroService.